modules/kibana/src/internalClusterTest/java/org/elasticsearch/kibana/KibanaThreadPoolIT.java

```diff
@@ -68,6 +65,7 @@ protected Settings nodeSettings(int nodeOrdinal, Settings otherSettings) {
             .put("thread_pool.write.queue_size", 1)
             .put("thread_pool.get.size", 1)
             .put("thread_pool.get.queue_size", 1)
+            .put(EnableAllocationDecider.CLUSTER_ROUTING_REBALANCE_ENABLE_SETTING.getKey(), EnableAllocationDecider.Rebalance.NONE)
```

Comment:
Can you please add a comment here why this is needed for future readers?

Reply:
👍 That would help lots

Reply:
++ see 491b1b774c81c5a2cbed0df053167d0e3ae42f0f